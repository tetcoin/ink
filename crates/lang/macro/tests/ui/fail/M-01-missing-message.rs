use pro_lang as pro;

#[pro::contract]
mod missing_message {
    #[pro(storage)]
    pub struct MissingMessage {}

    impl MissingMessage {
        #[pro(constructor)]
        pub fn constructor() -> Self {
            Self {}
        }
    }
}

fn main() {}
