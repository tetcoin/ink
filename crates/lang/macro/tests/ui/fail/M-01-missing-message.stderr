error: missing pro! message
  --> $DIR/M-01-missing-message.rs:4:1
   |
4  | / mod missing_message {
5  | |     #[pro(storage)]
6  | |     pub struct MissingMessage {}
7  | |
...  |
13 | |     }
14 | | }
   | |_^
